### PR TITLE
Issue #1144 - Added ISSUE_TEMPLATE.md for new issue creation

### PR DESCRIPTION
**Pull request by ujjwalagrawal17**
fix: #717 created template for creating a new issue by contributors (what they observed and what should be instead of that, how others can reproduce the same issue like this, system log).

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+**Observed behavior:** 
+
+What did you see instead?  Describe your issue in detail here.
+
+**Device and Android version:** 
+
+What make and model device (e.g., Samsung J7) did you encounter this on?  What Android
+version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
+ the stock
+version from the manufacturer or a custom ROM ?
```
Comment (maskaravivek) [inline ISSUE_TEMPLATE.md]: App version will also be helpful. Check out this link for more info about how to file a bug report. https://www.mediawiki.org/wiki/How_to_report_a_bug

Comment (ujjwalagrawal17) [reply ISSUE_TEMPLATE.md]: Sir, I had added it. Can we merge it now?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Summarize your issue in one sentence (what goes wrong, what did you expect to happen)
+
+**Steps to reproduce:** 
+
+How can we reproduce the issue?
+
+**Add System logs:**
+
+Add logcat files here(If you are able to add).
```
Comment (misaochan) [inline ISSUE_TEMPLATE.md]: Suggestion: change this to "Add logcat files here (if possible)."

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (misaochan) [inline ISSUE_TEMPLATE.md]: Was this intended to be on a new line?

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+What did you see instead?  Describe your issue in detail here.
+
+**Device and Android version:** 
+
+What make and model device (e.g., Samsung J7) did you encounter this on?  What Android
+version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
+ the stock
+version from the manufacturer or a custom ROM ?
+What version of Commons app are you running?
```
Comment (misaochan) [inline ISSUE_TEMPLATE.md]: Could add `You can find this information by going to the navigation drawer in the app and tapping 'About'`

**Comment by misaochan** [comment]
Thanks for the PR! Happy to merge pending the suggested changes. :)

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (ujjwalagrawal17) [reply ISSUE_TEMPLATE.md]: Sure sir

**Comment by ujjwalagrawal17** [comment]
I had updated it again.

**Pull request merged**